internal/descriptor/descriptor.go

```diff
@@ -45,3 +46,15 @@ func FromOCI(desc ocispec.Descriptor) Descriptor {
 		Size:      desc.Size,
 	}
 }
+
+func IsForeignLayer(desc ocispec.Descriptor) bool {
```

Comment:
nit: Missing documentation for this method.

Reply:
Added the documentation.